### ISSUE TITLE
LPIRL: Redundant Constraints

### ISSUE THREAD

**Issue by aaronsnoswell** [yrlu/irl-imitation#4]
Hi! Thank you for this great reference implementation - it is very helpful.

I was going over the LPIRL implementation and I think you have some redundant constraints in your LP matrices - see [line 59 in lp_irl.py](https://github.com/stormmax/irl-imitation/blob/eb36ee28f863b67944011915ae14c21d9b55243a/lp_irl.py#L59) - this loop does the same thing as the previous loop on line 55, resulting in a redundant set of constraints.

Thanks again,

**Comment by MithunNallana** [comment]
@aaronsnoswell 

Yeah. I too think the same way. In line 60, It should be -1.0 instead of 1.0.
If I am correct, constraint in line 55 and line 59 should provide constraints that proxy l1 norm. Can you please check this @yrlu .
